src/main/resources/assets/computercraft/lua/rom/programs/mkdir.lua

```diff
@@ -5,5 +5,11 @@ if #tArgs < 1 then
 end
 
 local sNewDir = shell.resolve( tArgs[1] )
+
+if fs.exists( sNewDir ) then
```

Comment:
Change to "fs.exists(sNewDir) and not fs.isDir(sNewDir)"
It's perfectly okay to "make" a directory that already exists (it should silently succeed